be/src/formats/parquet/file_reader.cpp

```diff
@@ -519,11 +521,18 @@ Status FileReader::_exec_only_partition_scan(vectorized::ChunkPtr* chunk) {
 }
 
 const tparquet::ColumnMetaData* FileReader::_get_column_meta(const tparquet::RowGroup& row_group,
-                                                             const std::string& col_name) {
+                                                             const std::string& col_name,
+                                                             const vectorized::HdfsScannerContext* ctx) {
```

Comment:
Should only pass what it need instead of entire `HdfsScannerContext`

Reply:
OK. I'll fix that.